src/components/AvatarMenu/AvatarMenu.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import { Avatar, Menu, MenuItem, ListItemIcon, ListItemText, Divider } from '@mui/material';
+import type { PropsAvatarMenu } from './AvatarMenu.types';
+import { DEFAULT_AVATAR_MENU_ITEMS } from './AvatarMenu.constants';
+import { AvatarMenuRoot, AvatarContainer, TriangleIndicator } from './AvatarMenu.styles';
+import { useAvatarMenu } from './AvatarMenu.hook';
+
+export const AvatarMenu: React.FC<PropsAvatarMenu> = ({
+	avatarUrl,
+	username,
+	menuItems = DEFAULT_AVATAR_MENU_ITEMS,
+	avatarProps,
+	fullWidth = false,
+	'data-testid': testId = 'avatar-menu',
+}) => {
+	const {
+		isOpen,
+		anchorEl,
+		handleToggle,
+		handleClose,
+		menuId,
+		triggerId,
+		focusedIndex,
+		handleMenuKeyDown,
+		setFocusedIndex
+	} = useAvatarMenu();
+
+	const enabledItems = menuItems.filter(item => !item.disabled);
+	const enabledItemsCount = enabledItems.length;
+
+	return (
+		<>
+			<AvatarMenuRoot
+				id={triggerId}
+				aria-label={`Menú de usuario${username ? `: ${username}` : ''}`}
+				aria-controls={isOpen ? menuId : undefined}
+				aria-haspopup="true"
+				aria-expanded={isOpen ? 'true' : 'false'}
+				onClick={handleToggle}
+				data-testid={testId}
+				$fullWidth={fullWidth}
+				$isOpen={isOpen}
+			>
+				<AvatarContainer $fullWidth={fullWidth}>
+					<Avatar
+						src={avatarUrl}
+						alt={username ? `${username.toUpperCase()}` : ''}
+						{...avatarProps}
+					/>
+					<TriangleIndicator $isOpen={isOpen} />
+				</AvatarContainer>
+			</AvatarMenuRoot>
+
+
+			<Menu
+				id={menuId}
+				anchorEl={anchorEl}
+				open={isOpen}
+				onClose={handleClose}
+				disablePortal={false}
+				keepMounted={false}
+				disableScrollLock={true}
+				anchorOrigin={{
+					vertical: 'bottom',
+					horizontal: 'right',
+				}}
+				transformOrigin={{
+					vertical: 'top',
+					horizontal: 'right',
+				}}
+				slotProps={{
+					paper: {
+						sx: {
+							marginTop: '8px',
+							zIndex: 1300,
```

Comment:
You set marginTop: '8px' inside slotProps.paper.sx which is a hardcoded px value. Per styling rules, avoid raw px and prefer theme spacing tokens.

**Sugerencia:**
```typescript
Use theme spacing via sx callback, e.g. paper: { sx: (theme) => ({ mt: theme.spacing(1), zIndex: theme.zIndex.modal }) } or marginTop: theme.spacing(1).
```